samples/subsys/ipc/ipm_mcux/master/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(BOARD lpcxpresso54114_m4)
+include(ExternalProject)
+
+ExternalProject_Add(
```

Comment:
interesting, is this going to be the way we build multi-core images? Can we have an RFC and eventually document this somewhere and streamline it a bit?

Reply:
How about having 'remote' project be part of the main  project, so you end up with 

```
ipm_mcux/
     CMakeLists.txt
     prj.conf
      master/*.c
      remote/
          prj.conf
          CMakeLists.txt
```

Reply:
> interesting, is this going to be the way we build multi-core images? Can we have an RFC and eventually document this somewhere and streamline it a bit?

Do we have an issue related to multi-core images, I think this is the start of how we would want to do certain aspects of it.

Reply:
> How about having 'remote' project be part of the main project, so you end up with

I've made this change, makes sense to me.

Reply:
> Do we have an issue related to multi-core images, I think this is the start of how we would want to do certain aspects of it.

we have this https://github.com/zephyrproject-rtos/zephyr/issues/3066, but we need to expand the problem description.